### ISSUE TITLE
add pretrained models

### ISSUE THREAD

**Issue by kingfischer16** [kingfischer16/srcnn-tf2#5]
Create a `pretrained` folder with a library of models. First should be the SRCNN model as found in the paper, with matching benchmark scores. Possibly add ensemble models, models trained with other loss functions, augmented data, or alternative datasets (with scores as well).

**Comment by kingfischer16** [comment]
pretrained SRCNN model (basic plus some of the better variants) were added in the #2 PR.
